.travis.yml

```diff
@@ -0,0 +1,21 @@
+language: java
+
+jdk:
+  - oraclejdk7
+  - oraclejdk8
+
+install: mvn -f gson install -DskipTests=true
+script: mvn -f gson test
```

Comment:
I looked at it more after writing this, and didn't realize that each folder was a separate Maven-based project. With that, a simple parent pom that includes each folder as a module will probably be enough to allow everything to be built at once.